test/fuzzing/build.sh

```diff
@@ -0,0 +1,28 @@
+export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
```

Comment:
Missing shebang, please add.

```suggestion
#!/usr/bin/env sh

set -eu

export FUZZER_DIR=$SRC/linkerd2/test/fuzzing
```